ivy/functional/frontends/jax/random.py

```diff
@@ -283,3 +283,21 @@ def pareto(key, b, shape=None, dtype="float64"):
     e = -ivy.log(1 - uniform)
 
     return ivy.exp(e / b)
+
+
+@handle_jax_dtype
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes(
```

Comment:
Can we make it clear that only supported dtypes for jax is `float32` and `float64`

Reply:
We can simply use `with_supported_dtypes` decorator